src/applications/appeals/shared/utils/focus.js

```diff
@@ -95,6 +100,11 @@ export const focusRadioH3 = () => {
   }
 };
 
+export const focusH3 = () => {
+  scrollTo('topContentElement');
+  focusByOrder(['#main h3', defaultFocusSelector]);
+};
+
```

Comment:
Should this portion be part of #30064?

Reply:
It is, but I needed to add it to make this PR work. I'll update this branch and this change will disappear.

Reply:
Noice!